CONTRIBUTING.md

```diff
@@ -0,0 +1,159 @@
+# Contributing
+
+Thank you for taking the time and effort to contribute to Greenstand!
+
+See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles
+them.
+Please make sure to read the relevant section before making your contribution.
+It will make project maintenance a lot easier and smooths out the experience for all involved.
+The community looks forward to your contributions.
+
+### Note
+
+Contributing to this project can sometimes involve a steep learning curve. Please do not give up and come and find us on
+Slack to get support setting you up. This is a fun project with an amazing potential to disrupt untransparent practices
+and open reforestation to the digital world.
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [Getting an issue assigned](#getting-an-issue-assigned)
+- [Working on an issue](#working-on-an-issue)
+- [Keeping your fork in sync](#keeping-your-fork-in-sync)
+- [Code style guide](#code-style-guide)
+- [Connect with Us](#connect-with-us)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Code of Conduct](./CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+
+## Getting an issue assigned
+
+- Look through the [open issues](https://github.com/Greenstand/treetracker-wallet-admin-client/issues) for one that
+  looks interesting.
+- Use the labels to help you find an issue:
+    - `good first issue`: easy and good for getting started.
+    - `medium`: medium difficulty or needs more work.
+    - `challenge`: hardest or big tasks, or needs some special skill or tricky or even hack in some way.
+    - `documentation`: writing job, sometimes it's good for new dev to learn and do some simple job.
+    - `bug`: just bug.
+    - `wontfix`: some issue still in discussion, or can not be implemented at current stage, or just outdated problem.
+    - `high-priority`: urgent problem, like some crucial bug or feature.
+    - We also tag issue with other aspects like the skill needed, the device related and so on.
+
+- If you're not sure what to work on, ask in the `#wallet-admin-client channel`
+  on [Slack](https://greenstand.slack.com/)
+  and we'll find a good issue for
+  you.
+- Add a comment to the selected issue to say you'd like to work on it, and ask for any clarification you need. Some of
+  the info you need to solve the problem may be missing from the description of the issue.
+- One of the Greenstand leads will then assign it to you and try to help with any questions.
+
+There are lots of opportunities to offer ideas and take ownership of larger pieces of work, so don't be afraid to ask!
+
+## Working on an issue
+
+- Create a branch for the issue in your local repo
+- Make your changes and test everything works locally
+- Push your changes to your fork on GitHub and create a pull request into Greenstand/main
```

Comment:
Currently, our target branch for Pull Requests is 'V1' and it will be 'V2' when we will work on the next version.